requirements.txt

```diff
@@ -1,7 +1,7 @@
 canonicalwebteam.flask-base==0.9.0
 alembic==1.5.4
 canonicalwebteam.http==1.0.3
-canonicalwebteam.blog==6.3.1
+canonicalwebteam.blog==6.4.0
```

Comment:
I don't understand, this is already the case on master.

Reply:
Weird. No idea how did made it here. I will rebase again.